static/includes/apps/CustomAppIntro.md

```diff
@@ -6,3 +6,16 @@ Generally, any container that follows the [Open Container Initiative](https://op
 To deploy a custom application, go to **Apps** and click **Discover Apps**.
 Click **Custom App** to open the **[Install iX App]({{< relref "installcustomappscreens.md #install-ix-app-screen" >}})** screen with a guided installation wizard.
 Click <i class="material-icons" aria-hidden="true" title="more_vert">more_vert</i> > **Install via YAML** to open the **[Custom App]({{< relref "installcustomappscreens.md #custom-app-screen" >}})** screen with an advanced YAML editor for deploying apps using Docker Compose.
+
+TrueNAS also allows users to convert already installed apps into custom applications. 
+Click on the app to wish to convert, and then click the widget next to **Edit** to open the menu containing **Update** and **Convert to custom app** options.
+
+{{< trueimage src="/images/SCALE/Apps/ConvertToCustomAppMenu.png" alt="Convert to Custom App Menu" id="Convert to Custom App Menu" >}}
+
+After selecting **Convert to custom app** from the menu, users can confirm the conversion on a confirmation screen.
```

Comment:
Style guide says to use dropdown list and not menu. It is okay to talk about the confirmation dialog or leave it out and shorten it by just saying select Confirm. How about this for clarity:
```suggestion
Select **Convert to custom app** from the dropdown list, then select **Confirm** to convert the app. 
```